packages/sso-button-react/package.json

```diff
@@ -42,5 +42,8 @@
   },
   "sideEffects": [
     "**/*.css"
-  ]
+  ],
+  "dependencies": {
+    "classnames": "^2.5.1"
```

Comment:
I added this tiny package for class management. https://www.npmjs.com/package/classnames